command/ca/provisionerbeta/update.go

```diff
@@ -192,6 +192,10 @@ provisioning tokens.`,
 			azureTenantFlag,
 			azureResourceGroupFlag,
 			removeAzureResourceGroupFlag,
+			azureSubscriptionIDFlag,
+			removeAzureSubscriptionIDFlag,
+			azureObjectIDFlag,
+			removeAzureObjectIDFlag,
```

Comment:
I don't see these flags taken into account in the `updateAzureDetails` function.